test/v1/connectors/RetrieveOtherReliefsConnectorSpec.scala

```diff
@@ -36,17 +35,36 @@ class RetrieveOtherReliefsConnectorSpec extends ConnectorSpec {
       appConfig = mockAppConfig
     )
 
+    val taxYear: TaxYear
+
+    val request: RetrieveOtherReliefsRequest = RetrieveOtherReliefsRequest(Nino(nino), taxYear)
+
+    val response = RetrieveOtherReliefsResponse(submittedOn = "2022-01-01", None, None, None, None, None, None, None)
   }
 
   "retrieve" should {
     "return a result" when {
-      val request: RetrieveOtherReliefsRequest = RetrieveOtherReliefsRequest(Nino(nino), TaxYear.fromMtd(taxYear))
 
       "the downstream call is successful" in new IfsTest with Test {
-        val outcome = Right(ResponseWrapper(correlationId, RetrieveOtherReliefsResponse))
+        lazy val taxYear: TaxYear = TaxYear.fromMtd("2017-18")
+
+        val outcome               = Right(ResponseWrapper(correlationId, response))
+
+        willGet(
+          url = s"$baseUrl/income-tax/reliefs/other/$nino/2017-18"
+        )
+          .returns(Future.successful(outcome))
+
+        await(connector.retrieve(request)) shouldBe outcome
+      }
+
+      "the downstream call is successful (TYS)" in new TysIfsTest with Test {
+        lazy val taxYear: TaxYear = TaxYear.fromMtd("2023-24")
+
+        val outcome               = Right(ResponseWrapper(correlationId, response))
```

Comment:
Can we refactor these tests to be like [these](https://github.com/hmrc/individuals-income-received-api/blob/main/test/v1/connectors/DeleteOtherCgtConnectorSpec.scala)?

Reply:
In what way? (The only difference I see is that the small `Test` trait is at the end rather than the start. Happy to move it but I'm always wary of adding cosmetic changes that might distract from the main purpose of the PR.)

Reply:
more in terms of the wording of the test cases rather than the tests themselves

Reply:
Understood. Will do.